setup.py

```diff
@@ -70,6 +70,10 @@
     "types-setuptools",
 ]
 
+legacy_keyring_dependencies = [
+    "keyrings.cryptfile @ git+https://github.com/frispete/keyrings.cryptfile@master",
```

Comment:
it makes me nervous to depend on `master` rather than a specific commit

Reply:
This is still a draft for that reason :)